CHANGELOG.md

```diff
@@ -5,6 +5,58 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [3.1.0] 2022-01-12
+
+The 3.1.0 release is a minor release for the plugin that does not introduce any breaking changes coming from the 3.0 branch. This version's primary focus is adding support for WordPress decoupled architectures and a revamped settings page. We have also worked hard on refining our code, testing, and delivery process.
+
+The Parse.ly plugin now hooks into the WordPress REST API to provide the Parse.ly metadata inserted automatically in non-decoupled sites. No new endpoint is introduced -- a `parsely` field is now rendered in the tracked objects containing that data (e.g., `post` and `page`). Please, refer to the plugin's README file for more details. Note that the tracking script must be inserted manually in the decoupled front-end. This behavior can be disabled using a filter.
```

Comment:
The final sentence here reads as though the behaviour of manually inserting a tracking script can be disabled using a filter. I think the last two sentences need switching order.